unit-tests/src/test/java/org/eclipse/collections/impl/bag/sorted/mutable/UnmodifiableSortedBagTest.java

```diff
@@ -256,14 +256,14 @@ public void withOccurrences()
     @Test
     public void addOccurrences_throws()
     {
-        assertThrows(UnsupportedOperationException.class, super::addOccurrences_throws);
+        // Not applicable for Unmodifiable
```

Comment:
All the *_throws are effectively throws.. IllegalArgumentEx due to bad inputs, ie add negative occurrence for example so I marked these as not applicable here.

We already capture the fact you can't modify an unmodifiable one in the non *_throws equivalents of these tests.